.github/workflows/workflow_rockcraft_cache_test.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Workflow Self-Test (Rockcraft Cache)
+
+on:
+  pull_request:
+  
+jobs:
+  save-cache:
+    uses: ./.github/workflows/build_rocks.yaml
+    secrets: inherit
+    with:
+      owner: ${{ github.repository_owner }}
+      registry: ghcr.io
+      working-directory: "tests/workflows/integration/test-indico-rock/"
```

Comment:
Is it possible to checkout this repository instead of copying the files?

Reply:
That's kind difficult since the `build_rocks` is a reusable workflow instead of an action.

Reply:
Yes, but this workflow job doesn't have steps.

Reply:
Noted